k8s-daemonset/k8s/linkerd-namerd.yml

```diff
@@ -28,14 +28,10 @@ data:
 
     - protocol: http
       label: incoming
-      dstPrefix: /
-      identifier:
-        kind: io.l5d.header
-        header: l5d-dst-concrete
       interpreter:
         kind: io.l5d.namerd
         dst: /$/inet/namerd.default.svc.cluster.local/4100
-        namespace: incoming
+        namespace: outgoing
```

Comment:
We should also rename this namespace since it's no longer specific to the outgoing router.